### PR TITLE
SED-1669 support simplified check syntax

### PR DESCRIPTION
**Pull request by david-stephan**

### CODE REVIEW

File: step-plans/step-plans-parser/src/main/java/step/repositories/parser/annotated/DefaultDescriptionStepParser.java [hunk 1]
```diff
@@ -440,7 +440,7 @@ public static void echo(ParsingContext parsingContext, String expression) {
 		parsingContext.addArtefactToCurrentParent(result);
 	}
 	
-	@Step("Check (.*)$")
+	@Step(value="Check (Expression *=.*)$", priority=2)
```
Comment (jeromecomte) [inline step-plans/step-plans-parser/src/main/java/step/repositories/parser/annotated/DefaultDescriptionStepParser.java:443]: - This expression requires exactly one space between "Check" and "Expression" and might break some existing plans having more than one space.
- Keyvalue.g4 considers additional chars as space: [ \u00A0\t\r\n]. In order not to break existing expression I would also allow them as space

Would therefore recommend: Check[ \u00A0\t\r\n]+(Expression[ \u00A0\t\r\n]=.*)$

File: step-plans/step-plans-parser/src/test/java/step/repositories/parser/DefaultDescriptionStepParserTest.java [hunk 2]
```diff
@@ -483,14 +484,30 @@ public void testIf() throws ParsingException {
...
 	@Test
 	public void testCheck() throws ParsingException {
 		List<AbstractStep> steps = new ArrayList<>();
-		steps.add(step("Check Expression= true \n"));
-
-		Check artefact = parseAndGetUniqueChild(steps, Check.class);
-		Assert.assertNotNull(artefact);
-		Assert.assertEquals("true",artefact.getExpression().getExpression());
+		steps.add(step("Check Expression= true \n "));
+		steps.add(step("Check Expression = key2.toString().equals('value2') \n "));
```
Comment (jeromecomte) [inline step-plans/step-plans-parser/src/test/java/step/repositories/parser/DefaultDescriptionStepParserTest.java:492]: Would add a test to cover the case with more than one space between Check and Expression: "Check   Expression=true"

**Review by jeromecomte — CHANGES_REQUESTED** [review]
See line comments

File: step-plans/step-plans-parser/src/main/java/step/repositories/parser/annotated/DefaultDescriptionStepParser.java (hunk 1 above)
Comment (david-stephan) [reply step-plans/step-plans-parser/src/main/java/step/repositories/parser/annotated/DefaultDescriptionStepParser.java]: Thanks, this indeed would have caused regressions, I updated as suggested.

File: step-plans/step-plans-parser/src/test/java/step/repositories/parser/DefaultDescriptionStepParserTest.java (hunk 2 above)
Comment (david-stephan) [reply step-plans/step-plans-parser/src/test/java/step/repositories/parser/DefaultDescriptionStepParserTest.java:492]: Added more test to cover more spaces but also non breaking space, endlines ...

**Review by jeromecomte — APPROVED** [review]

**Pull request merged**